test/test_buddyinfo.cpp

```diff
@@ -29,6 +29,17 @@ TEST_CASE("Parse buddyinfo", "[procfs][buddyinfo]")
         expected.chunks  = {1, 1, 1, 0, 2, 1, 1, 0, 1, 1, 3};
     }
 
+    // 9 chunks instead of the default of 11
+    SECTION("Sample 3")
```

Comment:
Can we please rename the tests to:
"Default size - Sample 1"
"Default size - Sample 2"
"Custom size - Smaller than default"